From 0c2f862fe23dc6d2c0ca8432d1f6027c922c5a04 Mon Sep 17 00:00:00 2001
From: Lev Stipakov <lev@openvpn.net>
Date: Tue, 11 Jan 2022 14:24:45 +0200
Subject: [PATCH] pkcs11.h: rename "interface" parameter

"interface" is defined in cobaseapi.h as

  #define interface __STRUCT__

so use different name.

Signed-off-by: Lev Stipakov <lev@openvpn.net>
---
 include/pkcs11-helper-1.0/pkcs11.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/pkcs11-helper-1.0/pkcs11.h b/include/pkcs11-helper-1.0/pkcs11.h
index 85aa98e..7a7b958 100644
--- a/include/pkcs11-helper-1.0/pkcs11.h
+++ b/include/pkcs11-helper-1.0/pkcs11.h
@@ -1210,7 +1210,7 @@ _CK_DECLARE_FUNCTION (C_GetInterfaceList,
 _CK_DECLARE_FUNCTION (C_GetInterface,
 		      (unsigned char *interface_name,
 		       struct ck_version *version,
-		       struct ck_interface **interface,
+		       struct ck_interface **iface,
 		       ck_flags_t flags));
 
 _CK_DECLARE_FUNCTION (C_LoginUser,
-- 
2.23.0.windows.1

